src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -397,15 +399,6 @@ private void hutDecClicked(@NotNull final Button button)
      */
     private List<String> getStyles()
     {
-        try
```

Comment:
Does he still load the file onTheFly or do we want to relog to get the newest styles if we drop them off?